tests/components/SLink.spec.ts

```diff
@@ -15,16 +16,17 @@ describe('components/SLink', () => {
       }
     })
 
-    expect(wrapper.find('router-link').exists()).toBe(true)
+    expect(wrapper.findComponent(RouterLink).exists()).toBe(true)
   })
 
-  test('creates `a` tag when link is external', () => {
+  test('creates `a` tag without RouterLink when link is external', () => {
     const wrapper = mount(SLink, {
       props: {
         href: 'https://example.com'
       }
     })
 
+    expect(wrapper.findComponent(RouterLink).exists()).toBe(false)
     expect(wrapper.find('a').exists()).toBe(true)
   })
```

Comment:
these changes are needed because router is now being injected via setup _for all pages_

RouterLink get's rendered so we can no longer find `router-link` as it will rendered to `<a ...>`. Hence using findComponent instead.

Also, this means that the tests testing that `a` gets created will pass in all cases. We need to test that anchor is getting rendered without `RouterLink` when link is or is marked as external.